docs/common-bugs.rst

```diff
@@ -322,6 +322,16 @@ Rotated screen
 :Solution: It seems to be a problem with drivers. Reassigning to kernel.
 :Contact: kernel or iio-sensor-proxy
 
+No video output with the MGA G200e graphics card
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:Issue: There is no video output with MGA G200e graphic card and a 1920x1080 monitor.
+:Solution: Add ``vga=795`` to the boot options before installation.
+    Alternatively it is also possible to enter the "Troubleshooting" menu when the
+    installation image boots and install in basic graphics mode.
+    Please note that the system will boot into text mode after installation in basic graphics mode.
```

Comment:
Maybe specify that it's the installed/resulting system.

Reply:
Lets do that!